package.nls.json

```diff
@@ -136,6 +136,8 @@
 	"githubIssues.assignWhenWorking.description": "Assigns the issue you're working on to you. Only applies when the issue you're working on is in a repo you currently have open.",
 	"githubPullRequests.focusedMode.description": "The layout to use when a pull request is checked out. Set to false to prevent layout changes.",
 	"githubPullRequests.showPullRequestNumberInTree.description": "Shows the pull request number in the tree view.",
+	"githubPullRequests.pullRequestLabels.description": "Group of labels that you want to add to the PR automatically",
+	"githubPullRequests.pullRequestLabels.label.description": "Each string element is value of label that you want to add",
```

Comment:
It would be good to also add: "Labels that don't exist in the repository won't be added".

Reply:
Done